### ISSUE TITLE
Sometimes the interval never stops.

### ISSUE THREAD

**Issue by Domvel** [wbhob/ionic-long-press#4]
Sometimes the interval / onPressing event never stops. Because unsecure button events. You have to handle press- cancle / abort event. e.g. when swiped the finger from button out the screen. (like pointercancel) Maybe it's also better to use setTimeout instead of setInterval, (restart timeout).

I don't know the ionic gesture events. It's just a string (event name) without a documentation. But the following works: (native events)

```ts
    this.el.addEventListener('pointercancel', (e: any) => {
        // call the same as on pressup.
    });

    this.el.addEventListener('pointerout', (e: any) => {
        // call the same as on pressup.
    });
```
Not the best solution. Because the pointerout events is thrown sometimes without a expected reason.

**Comment by wbhob** [comment]
I don't really know how to fix this. If you make a PR, I will merge it.

**Comment by jcphlux** [comment]
Was there a PR done for this?

**Comment by wbhob** [comment]
Not that I know of, haven't had any PRs for a while.

**Comment by jcphlux** [comment]
ok I will see if I can get it working and if so I will submit a PR

**Comment by jcphlux** [comment]
So I was able to solve this partially. if you move your finger off the button I can get it to stop firing unless you move your finger very fast or very slow.

**Comment by wbhob** [comment]
Fixed in #6
